packages/webpack/mixin.core.js

```diff
@@ -11,17 +11,28 @@ const {
 
 const { Mixin } = require('@untool/core');
 
+const { Resolvable } = require('./lib/utils/resolvable');
+
 class WebpackMixin extends Mixin {
-  createAssetsMiddleware() {
-    const assetsMiddleware = require('./lib/middleware/assets');
-    const { config, assets, assetsByChunkName } = this;
-    return assetsMiddleware(config, { assets, assetsByChunkName });
+  constructor(...args) {
+    super(...args);
+    this.assets = new Resolvable();
+  }
+  loadPrebuiltAssets() {
+    const { serverDir, assetFile } = this.config;
+    const path = join(serverDir, assetFile);
+    return exists(path) ? require(path) : {};
```

Comment:
Should we print a warning in case the `assetFile` does not exist?

Reply:
I have to think this through - too many cases...